src/plasoscaffolder/bll/mappings/mapping_helper.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Helper methods for mapping"""
+import os
+
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+
+
+def _get_template_path() -> os.path:
+  """
+  the path to the template file
```

Comment:
This is a function to there is an action.

Retrieves the path to the template file.